lib/cudnn/src/libcudnn.jl

```diff
@@ -2927,51 +2927,51 @@ end
 end
 
 @checked function cudnnSetConvolution2dDescriptor(convDesc, pad_h, pad_w, u, v, dilation_h,
-                                                  dilation_w, mode, computeType)
```

Comment:
These headers are auto-generated, so you shouldn't edit them.

Reply:
Can you point me to where they were generated?
The naming was confusing to me when trying to understand how to use the API.
From my commit message:
> Not sure why it's called compute type here, but the C-side of the API
also just calls it `dataType` (see [here](https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnGetConvolutionNdDescriptor)).

Reply:
It's called like that in the CUDNN headers:

```
/usr/include/cudnn_cnn_infer.h
130:cudnnSetConvolution2dDescriptor(cudnnConvolutionDescriptor_t convDesc,
131-                                int pad_h,      /* zero-padding height */
132-                                int pad_w,      /* zero-padding width */
133-                                int u,          /* vertical filter stride */
134-                                int v,          /* horizontal filter stride */
135-                                int dilation_h, /* filter dilation in the vertical dimension */
136-                                int dilation_w, /* filter dilation in the horizontal dimension */
137-                                cudnnConvolutionMode_t mode,
138-                                cudnnDataType_t computeType);
```

The wrapper generator scripts are in `res/`, but I don't think it's worth carrying a patch or rewriting functionality to change a function argument name. So I'd suggest removing the change and leaving it be, even if it's a little unclear.

Reply:
Done.